src/test_server.py

```diff
@@ -41,7 +28,7 @@ def test_response_ok_one():
 def test_response_ok_two():
     """Test response_ok with specific test data."""
     temp = response_ok()
-    header_lines, body = parse_request(temp)
+    header_lines, body = parse_message(temp)
     temp2 = header_lines[0].split()
     assert temp2[0] == "HTTP/1.1"
     assert temp2[1] == "200"
```

Comment:
You should only have one assert statement per test for testing this kind of thing. The previous three lines are all setup which you can do in a fixture which returns e.g. a tuple of the 3 parts of your first line. The point of this is so that you are testing the correctness of each component isolated. If the first assert fails, the test function will exit and you will not know whether the next two failed or not.